driver/driver.go

```diff
@@ -19,6 +19,8 @@ const (
 	// Null driver represents an empty driver for use by benchmarks that
 	// require no driver
 	Null
+	//CRI driver represents k8s Container Runtime Interface
+	CRI
```

Comment:
can you add this above `Null` so that `Null` is the last driver in the list? Not a big deal, but.. :)